package.json

```diff
@@ -50,6 +50,58 @@
       "ecmaVersion": 2019
     }
   },
+  "husky": {
+    "hooks": {
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
+    }
+  },
+  "commitlint": {
+    "rules": {
+      "body-leading-blank": [
+        2,
+        "always"
+      ],
+      "body-max-line-length": [
+        2,
+        "always",
+        72
+      ],
+      "body-min-length": [
+        2,
+        "always",
+        20
+      ],
+      "scope-empty": [
+        2,
+        "always"
```

Comment:
Yep, that was intended.  To me, this package is *only* the logger.  There may be a few other libs for things like backoff strategy, but I felt it was easier to just disallow it rather than try to validate the possible values for the scope.  If you feel strongly about having scope, shoot some options for allowable values?

Reply:
nope no strong feelings here and that reasoning makes sense to me, just double checking 👍